moped-editor/src/views/dashboard/DashboardTimelineModal.js

```diff
@@ -58,10 +77,32 @@ const DashboardTimelineModal = ({
           </IconButton>
         </DialogTitle>
         <DialogContent>
-          <ProjectTimeline
-            table={table}
-            projectIdFromTimelineModal={projectId}
-          />
+          <ApolloErrorHandler error={error}>
+            <CardContent>
+              <Grid container spacing={2}>
+                <Grid item xs={12}>
+                  <Box mb={2} style={{ maxWidth: "100%" }}>
+                    {table === "phases" && (
```

Comment:
![chefs-kiss-french-chef](https://user-images.githubusercontent.com/5697474/188687927-b2179416-718f-4ec8-ae11-9a61fffcca62.gif)

Reply:
> Love this refactor. So much more organized and reusable! Thanks for having the patience and persistence on this. I didn't test the UI and I know you were working on a few last bug fixes, but just wanted to give positive feedback on the code. 👍

thanks @mateoclarke!! :) all the bug fixes are in the pr so it should be working smoothly! thanks for the review, hope all is going well!